### ISSUE TITLE
OpenGL bindless texture crash in driver 24.10.1

### ISSUE THREAD

**Issue by UltraEngine** [GPUOpen-Drivers/AMD-Gfx-Drivers#27]
In driver 24.10.1 it seems that calls to textureQueryLevels and textureSize using samplers from bindless texture handles cause a crash. No OpenGL errors are reported. My previous driver version 24.7.1 had no such problems with the same code.

Example: https://www.ultraengine.com/files/AMDBindlessTexturesBug.zip

In Shaders/PBR/Lighting.glsl:
```
u_MipCount = textureQueryLevels(samplerCube(specularmap));
```
In Shaders/Base/Lighting.glsl:
```
float sz = 1.0f / float(textureSize(shadowmap, 0).x);
```
Note that the code doesn't even get executed, but its presence in the shader is enough to cause the crash.

First reported here: https://www.ultraengine.com/community/topic/66998-editor-crashing-on-amd-radeon-drivers-beyond-2481/

**Comment by baozholu-AMD** [comment]
Thanks for reporting it, an internal bug ticket has been created to track this issue, will keep you posted on its progress. Thanks.

**Comment by UltraEngine** [comment]
Thank you. Is this the best channel for reporting driver bugs? AMD's drivers have been so stable the last few years, that it has been a long time since I needed to report anything.

**Comment by baozholu-AMD** [comment]
> Thank you. Is this the best channel for reporting driver bugs? AMD's drivers have been so stable the last few years, that it has been a long time since I needed to report anything.

Yes, we pay close attention to and resolve issues reported to this forum.

**Comment by baozholu-AMD** [comment]
This issue is understood and fixed internally. The future Adrenalin driver releases will include the complete fix.

**Comment by UltraEngine** [comment]
The issue does not seem to be resolved when testing my application above with this configuration:
GPU: RX 6600
Driver: 24.12.1
OS: Windows 10 64-bit

**Comment by baozholu-AMD** [comment]
@UltraEngine This fixed not yet included into the latest 24.12.1 Adrenalin driver. Appreciate your patience for the moment, thanks.

**Comment by UltraEngine** [comment]
Considering the long turnaround time on this, are one of the following options possible?

- Allow me access to test a beta version of the next driver.
- I send you a license for our software product, for internal testing.

The driver team tested a small application I uploaded to demonstrate the issue, but no one has tested the full application yet.
